WoWsShipBuilder.UI/ViewModels/ShipSelectionWindowViewModel.cs

```diff
@@ -34,6 +36,13 @@ public ShipSelectionWindowViewModel(Window? win)
             Dictionary<string, ShipSummary> shipNameDictionary = AppData.ShipSummaryList.ToDictionary(ship => Localizer.Instance[$"{ship.Index}_FULL"].Localization, ship => ship);
             FilteredShipNameDictionary = new SortedDictionary<string, ShipSummary>(shipNameDictionary);
             SummaryList = new AvaloniaList<KeyValuePair<string, ShipSummary>>(FilteredShipNameDictionary.Select(entry => entry));
+            multiSelectionEnabled = multiSelection;
+            SelectedShipList.CollectionChanged += SelectedShipList_CollectionChanged;
```

Comment:
Make sure that this will not cause memory leaks

Reply:
The whole window object is disposed after being closed, so it should not cause any memory leak (the event itself is implemented with weakEvents)